tests/test_license.py

```diff
@@ -107,9 +107,20 @@ def setUp(self):
             'src/opendr/perception/object_detection_2d/detr/algorithm',
             'src/opendr/perception/object_detection_2d/nanodet/algorithm',
             'src/opendr/perception/panoptic_segmentation/efficient_ps/algorithm/EfficientPS',
+            'src/opendr/perception/facial_expression_recognition/landmark_based_facial_expression_recognition',
             'src/opendr/perception/facial_expression_recognition/landmark_based_facial_expression_recognition/algorithm',
```

Comment:
I am not sure about this, it appeared during the merge with fer-va PR, should probably keep only the first, more general, one.